lib/utils.js

```diff
@@ -50,6 +50,7 @@ class FreshdeskError extends Error {
 
 		this.apiTarget = `${req.method} ${req.path}`;
 		this.requestId = res.headers["x-request-id"];
+		this.retryAfter = parseInt(res.headers["retry-after"]);
```

Comment:
is this header always present? do we need `null` check?

Reply:
we don't need null check since `parseInt` will return NaN if this header is undefined.